frappe/query_builder/functions.py

```diff
@@ -2,6 +2,8 @@
 from pypika.terms import Function
```

Comment:
Let's explicitly import `Min`, `Max`, `Avg`, and `Sum` from `pypika.functions`

to avoid sider warnings.

Reply:
I'd chalk that as an issue with sider, and thus something we shouldn't have to cater for. Since even your text editor can point out easily where it's actually defined. 🤔

Reply:
agreed but do we really need all the functions to be imported in this file?

Reply:
They are imported so that users can write `from frappe.query_builder.functions import ...`. The functions defined here override the ones from pypika's module :)